lib/http.js

```diff
@@ -40,10 +39,7 @@ exports.get = function(options, cb) {
 exports._connectionListener = server._connectionListener;
 const Server = exports.Server = server.Server;
 
-exports.createServer = function(requestListener) {
-  return new Server(requestListener);
-};
-
+exports.createServer = requestListener => new Server(requestListener);
```

Comment:
I'm perfectly fine with either (both are readable IMO) but I definitely something that thing should be specified clearly in a coding styleguide.